src/script/Deployment/Publisher/start.sh

```diff
@@ -34,6 +34,10 @@ check_link(){
   fi
 }
 
+#directories/files that should exist before creating links (SERVICE is type of Publisher started, e.g.Publisher_rucio):
```

Comment:
so they should be created before starting the container, correct ?

Reply:
based on my previous answer/idea, they should exits before starting container. And I implemented check for that in container start script (will push changes a bit later today). I am fully open to move _'directory exists'_ checks to TW/Publisher start script :)

Reply:
so I'd change comment to say so. Can commit directly to the master branch (e.g. edit in GH) w/o bothering with a PR. If you prefer that I do it, tell me and I will.

Reply:
Done.